bin/varnishd/cache/cache_acceptor.c

```diff
@@ -516,7 +516,7 @@ vca_accept_task(struct worker *wrk, void *arg)
 
 		wa.acceptsock = i;
 
-		if (!Pool_Task_Arg(wrk, TASK_QUEUE_VCA,
+		if (!Pool_Task_Arg(wrk, TASK_QUEUE_REQ,
```

Comment:
wouldn't the failure case a few lines down need the same adjustment?

Reply:
No, the one below is letting the incoming session take over the thread and rescheduling the acceptor, so only this one is incorrect.

Reply:
right, thank you